Makefile

```diff
@@ -31,6 +31,7 @@ RELEASE_CONTROLLER_IMAGE:=controller:$(RELEASE_VERSION)
 # v20201009-v0.18.800-46-g939c1c0 - automated build for a commit(not a tag) and also a local build
 # v20200521-v0.18.800             - automated build for a tag
 VERSION=$(shell echo $(RELEASE_VERSION) | awk -F - '{print $$2}')
+VERSION:=$(or $(VERSION),v0.0.0)
```

Comment:
I agree a fix like this is beneficial for the project because it lowers the barrier to consume and experiment with.
Thinking about it, for internal snapshots/tests in RH we use the format `v0.0.$(date +%Y%m%d)` (we actually append also a sequential, but we can defer that) so it reads like `v0.0.20230324` for example.

I think it's worth mentioning but I have no real preference here.

Reply:
sound great, applied the change to make it date based